ndk-build/src/manifest.rs

```diff
@@ -258,6 +262,39 @@ pub struct Permission {
     pub max_sdk_version: Option<u32>,
 }
 
+/// Android [package element](https://developer.android.com/guide/topics/manifest/queries-element#package).
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+pub struct Package {
+    #[serde(rename(serialize = "android:name"))]
+    pub name: String,
+}
+
+/// Android [provider element](https://developer.android.com/guide/topics/manifest/queries-element#provider).
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+pub struct QueryProvider {
+    #[serde(rename(serialize = "android:authorities"))]
+    pub authorities: String,
+
+    // The specs say only `authorities` attribute is the only one needed for providers contained in `queries`
+    // however aapt throws errors requireing a `android:name` attribute, nd-build should be using appt2?
+    #[serde(rename(serialize = "android:name"))]
+    pub name: String,
+}
+
+/// Android [queries element](https://developer.android.com/guide/topics/manifest/queries-element)
```

Comment:
```suggestion
/// Android [queries element](https://developer.android.com/guide/topics/manifest/queries-element).
```

Stay consistent with the rest of your change (and the doc comments here in general).